index.php

```diff
@@ -32,6 +68,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Top Cat</title>
     <link rel="stylesheet" href="style.css">
+    <script defer src="js/script.js"></script>
```

Comment:
https://www.frontiertouring.com/web_images/1603/merchant_1603.jpg
This filename should be clearer. What's it doing?